colorscad.sh

```diff
@@ -120,14 +126,14 @@ if [ "$FORMAT" != amf ] && [ "$FORMAT" != 3mf ]; then
 	exit 1
 fi
 
-if ! command -v openscad &> /dev/null; then
-	echo "Error: openscad command not found! Make sure it's in your PATH."
+if ! command -v ${OPENSCAD_CMD} &> /dev/null; then
```

Comment:
Shellcheck wants everything to be quoted:
```suggestion
if ! command -v "$OPENSCAD_CMD" &> /dev/null; then
```